### ISSUE TITLE
Cannot build github.com/go-llsqlite/crawshaw

### ISSUE THREAD

**Issue by izissise** [please-build/go-rules#247]
When I try to build ` github.com/go-llsqlite/crawshaw`
I get the following error:
```
fatal error: ./c/sqlite3.c: No such file or directory
   20 | // #include "./c/sqlite3.c"
      |           ^~~~~~~~~~~~~~~
compilation terminated.
```

It looks like please_go_tool doesn't correctly adds `c/sqlite3.c` in `c_srcs`

```
...
    c_srcs = [
        "blocking_step.c",
        "wrappers.c",
    ],
...
```

**Comment by peterebden** [comment]
Yeah honestly this looks pretty degenerate as far as we're concerned - it includes a .c file from another Go package, which isn't something we typically make available. I'm not sure what information is available to us to detect that we should do that (or even exactly what we'd do here; given the `c` directory is itself a package we can't directly pull in source files from it)

**Comment by izissise** [comment]
For this case just adding `c/sqlite3.c` in `c_srcs` makes it compile. 
Maybe add a parameters to please_go_tool that pre-include `c_srcs` or `go_src`?
